doctypes.tex

```diff
@@ -1,19 +1,67 @@
 \subsection{Types of Documentation}
+\label{sec:doc_types}
+
+Data Management creates a variety of documentation for different audiences, and with different purposes.
+Broadly, we break documentation efforts into two categories: project documentation and user documentation.
+Project documentation is a tree of requirements documents, interface control documents, design documents, and test reports that aides the construction project.
+User documentation, on the other hand, is a deliverable that we ship alongside our software and services to either the scientific community or future developers and operators.
 
 \subsubsection{Project documentation}
+\label{sec:project_docs}
 
-* Controlled
+\paragraph{Documentation tree}
 
-* Technical notes
+DM creates project documentation that fits within LSST's overall Document Management Plan.\cite{LPM-51}
+As is standard practice, we organize the collection of project documents in a tree.
+LSST Project Management and Systems Engineering requirements documents form the tree's trunk, along with interface control documents that DM collaborates on with other LSST subsystems.
+These flow into Data Management requirements documents for each DM product: Database,\cite{LDM-555} Middleware,\cite{LDM-556} Science Platform,\cite{LDM-554} and Data Management System Design.\cite{LDM-148}
+From these, we maintain design documents: Database Design,\cite{LDM-135} Middleware Design,\cite{LDM-152} Science Platform Design,\cite{LDM-542} Services and Infrastructure Design,\cite{LDM-129} Pipeline Design,\cite{LDM-151} and Network Design.\cite{LSE-78}
```

Comment:
We are probably going to have to remove half of this paragraph for space reasons but I'm happy to delay it until an editing phase.